transforms/StickbreakingPowerLogistic/StickbreakingPowerLogistic_functions.stan

```diff
@@ -0,0 +1,36 @@
+vector stickbreaking_power_logistic_simplex_constrain_lp(vector y) {
+  int N = rows(y) + 1;
+  vector[N] x;
+  real log_u, log_w, log_z;
+  real log_cum_prod = 0;
+  for (i in 1 : (N - 1)) {
+    log_u = log_inv_logit(y[i]); // logistic_lcdf(y[i] | 0, 1);
+    log_w = log_u / (N - i);
+    log_z = log1m_exp(log_w);
+    x[i] = exp(log_cum_prod + log_z);
+    target += 2 * log_u - y[i]; // logistic_lupdf(y[i] | 0, 1);
```

Comment:
This isn't necessarily going to be equal to the `logistic_lupdf`.  All the `_lupdf` function guarantees is equality up to an additive constant---we don't say exactly what's going to get dropped.

Reply:
Ah, thanks, that's actually an incorrect comment (should be `_lpdf`)